trademarks.md

```diff
@@ -0,0 +1,144 @@
+# Trademark Usage Policy
+
+**The following is a draft of the Jupyter Trademark Usage Policy. We are actively working on this document.
+Sections that are completed should be considered to be accepted and active.**
+
+
+## Introduction
+
+This document outlines the policy of the Project Jupyter ("Jupyter") regarding the use of its trademarks.
+Any use of any Project Jupyter trademark must be in accordance with this policy.
+
+"Jupyter" is a trademark of the NumFOCUS foundation, of which Project Jupyter is a part.
+
+The Jupyter logos (in several variants) are trademarks of NumFOCUS as well.
+
+Registration of the trademarks is pending.
+
+Derivative word marks referring to Jupyter projects or events, such as "JupyterDays," "JupyterLab," "JupyterHub," "Jupyter Notebook" are also covered by this policy.
+
+All trademarks, even those that apply to open source software, must be used according to certain legal requirements. If these requirements are not met, the trademark may be endangered or lost. One of these requirements is for the trademark owner (in this case, NumFOCUS) to maintain standards for using its trademarks, and to enforce acceptable use of the trademarks by taking action against parties that violate those standards.
```

Comment:
Due to mixture of copy-paste from PSF, plus newly written content. Should be consistent now.